dashboard-for-web/articles/web-dashboard-designer-mode/appearance-customization/conditional-formatting.md

```diff
@@ -27,68 +27,87 @@ Comparison rules used in conditional formatting can be divided into the followin
 * **Bar Color Ranges** - Allows you to visualize numeric values using bars whose colors are contained in the specified color set.
 * **Bar Gradient Ranges** - Allows you to visualize numeric values using bars whose colors are contained in the specified color gradient.
```

Comment:
Please see my similar comment above. If my understanding is correct, would it be technically correct to say:

Bar Gradient Ranges - Visualizes numeric values as bars. Values within a range display a specific bar gradient.

Reply:
```suggestion
* **Bar Gradient Ranges** - Visualizes numeric values as bars. Values within a range display a specific color.
```

Reply:
Change to:

* **Bar Gradient Ranges** - Visualize numeric values as bars. Values within a range display a specific color.